xep-0060.xml

```diff
@@ -243,7 +243,7 @@
     <version>1.0</version>
     <date>2003-10-28</date>
     <initials>psa</initials>
-    <remark><p>Per a vote of the Jabber Council, advanced status to Draft.</p></remark>
+    <remark><p>Per a vote of the XMPP Council, advanced status to Draft.</p></remark>
```

Comment:
I wouldn't change the release notes; at the time of writing it was still the "Jabber Council" (now it's the XSF Council).